File: Bai_69/main.py
import re
input_file = "abc.txt"
output_file = "number.txt"
with open(input_file, 'r', encoding='utf-8') as f:
    data = f.read()
    numbers = re.findall(r'\d+', data)
with open(output_file, 'w') as out:
    for number in numbers:
        out.write(number + '\n')

File: Bai_69/main.cpp
#include<iostream>
#include<regex>
#include<fstream>
using namespace std;
int main(){
    ifstream ifile("abc.txt");
    ofstream ofile("number.txt");
    string s = " ";
    string data((istreambuf_iterator<char>(ifile)), istreambuf_iterator<char>());
    regex pattern("\\d+");
    sregex_iterator next(data.begin(), data.end(), pattern);
    sregex_iterator end;
    while (next != end){
        ofile << (*next).str() << endl;
        ++next;
    }
    ifile.close();
    ofile.close();
    return 0;
}